src/frontend/js/widgets/Dashboard/components/ListFilterOrganization/index.tsx

```diff
@@ -0,0 +1,58 @@
+import { defineMessages, useIntl } from 'react-intl';
+import { Select, SelectProps } from '@openfun/cunningham-react';
+import { useEffect } from 'react';
+import { useOrganizations } from 'hooks/useOrganizations';
+import { Spinner } from 'components/Spinner';
+
+export const messages = defineMessages({
+  organizationFilterLabel: {
+    defaultMessage: 'Organization',
+    description: 'Use as organization filter label',
+    id: 'pages.TeacherDashboardContractsLayout.OrganizationContractFilter.organizationFilterLabel',
```

Comment:
use this id instead? `components.ListFilterOrganization.organizationFilterLabel`